mods/d2k/rules/defaults.yaml

```diff
@@ -446,18 +446,20 @@
 	Inherits: ^Building
 	Inherits@selection: ^SelectableCombatBuilding
 	WithSpriteTurret:
+		RequiresCondition: !build-incomplete
 	AttackTurreted:
 	RenderRangeCircle:
 	DetectCloaked:
 		Range: 1c768
 	-GivesBuildableArea:
-	-WithMakeAnimation:
 	-WithCrumbleOverlay:
 	-Capturable:
-	-WithSpriteBody:
+	WithSpriteBody:
+		Sequence: null
```

Comment:
This means the actor now has two sprite bodies without identifier (even if one of them is a `WithWallSpriteBody`), which even both have the internal default `Name: body`.

It would be cleaner and clearer to readers to change this to something like
```
	WithSpriteBody:
		Name: make-enabler
		Sequence: null
	WithMakeAnimation:
		BodyNames: make-enabler
```

Reply:
Good idea.

Reply:
I wonder why `CheckSpriteBodies` didn't trigger.